Specs/Cesium3DTilesTester.js

```diff
@@ -193,6 +197,7 @@ Cesium3DTilesTester.rejectsReadyPromiseOnError = function (
       fail("should not resolve");
     })
     .catch(function (error) {
+      console.log(error);
```

Comment:
Was this added for debugging purposes, or is this an intentional change?

Reply:
oops forgot to remove it :sweat_smile: